integrations/adobe-analytics/lib/index.js

```diff
@@ -418,50 +623,78 @@ AdobeAnalytics.prototype.checkoutStarted = function(track) {
  * @param {string} adobeEvent Adobe standard event
  */
 
-AdobeAnalytics.prototype.processEvent = function(msg, adobeEvent) {
-  var merchEvents = getMerchConfig(msg, this.options);
+AdobeAnalytics.prototype.processEvent = function (msg, adobeEvent) {
   var properties = msg.properties();
+  let adobeEvents = [];
```

Comment:
```suggestion
  var adobeEvents = [];
```
browser compatibility